server/lib/konnector_hash.coffee

```diff
@@ -2,7 +2,7 @@ fs = require 'fs'
 path = require 'path'
 
 currentPath = path.dirname fs.realpathSync __filename
-modulesPath = path.join currentPath, '..', 'konnectors'
+modulesPath = path.join currentPath, '../konnectors'
```

Comment:
Updated. This change was made previously, and at the time I was needing to put something between the `..` and the `konnectors` part, hence it was easier to just edit a single string, if I remember correctly. Should be reverted indeed.